yt_dlp/extractor/mave.py

```diff
@@ -0,0 +1,84 @@
+import json
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    get_element_by_id,
+)
+
+
+class MaveIE(InfoExtractor):
+    _VALID_URL = r'https?://(?P<channel_id>[a-z]+)\.mave\.digital/ep-(?P<short_id>[0-9]+)'
+    _TESTS = [
+        {
```

Comment:
you'll have an easier time generating & adding the test data if you format the `_TESTS` like this and unindent below:
```suggestion
    _TESTS = [{
```